lib/rules/accessor-pairs.js

```diff
@@ -66,80 +66,97 @@ function isPropertyDescriptor(node) {
 // Rule Definition
 //------------------------------------------------------------------------------
 
-module.exports = function(context) {
-    var config = context.options[0] || {};
-    var checkGetWithoutSet = config.getWithoutSet === true;
-    var checkSetWithoutGet = config.setWithoutGet !== false;
-
-    /**
-     * Checks a object expression to see if it has setter and getter both present or none.
-     * @param {ASTNode} node The node to check.
-     * @returns {void}
-     * @private
-     */
-    function checkLonelySetGet(node) {
-        var isSetPresent = false;
-        var isGetPresent = false;
-        var isDescriptor = isPropertyDescriptor(node);
-
-        for (var i = 0, end = node.properties.length; i < end; i++) {
-            var property = node.properties[i];
-
-            var propToCheck = "";
-            if (property.kind === "init") {
-                if (isDescriptor && !property.computed) {
-                    propToCheck = property.key.name;
-                }
-            } else {
-                propToCheck = property.kind;
+var GETTER_MISSING_MESSAGE = "Getter is not present",
```

Comment:
Mostly because I don't think people will use it. If we make them jump through hoops to get at their messages, they're just going to insert strings directly. 

I think we need to rethink this piece and possibly defer until later.